tests/integration/targets/meraki_ms_switchport/tasks/main.yml

```diff
@@ -724,3 +724,84 @@
     that:
       - idempotent_delete_mac.changed == False
       - idempotent_delete_mac.data is defined
+
+- name: Create access policy with auth_method is "my Radius Server"
+  cisco.meraki.meraki_ms_access_policies:
```

Comment:
I believe this would depend on merging #346 correct?

Reply:
Yup, but you can test it via creating a access policy manually